dev-tools/compose/docker-compose.discovery.prod.yml

```diff
@@ -12,8 +12,6 @@ services:
     deploy:
       mode: replicated
       replicas: '${DISCOVERY_PROVIDER_REPLICAS}'
-    ports:
-      - '6379:6379'
```

Comment:
does this break @schottra's fetch OTP script?

Reply:
actually I think this breaks solana-relay tests which we don't have in CI yet
https://github.com/AudiusProject/audius-protocol/blob/main/packages/discovery-provider/plugins/pedalboard/apps/solana-relay/src/routes/listen/assertListenRateLimiter.test.ts#L6-L18